app/models/supervisor_volunteer.rb

```diff
@@ -4,6 +4,12 @@ class SupervisorVolunteer < ApplicationRecord
   belongs_to :volunteer, class_name: "User"
   belongs_to :supervisor, class_name: "User"
   validates :supervisor_id, uniqueness: {scope: :volunteer_id} # only 1 row allowed per supervisor-volunteer pair
+  validates :volunteer_id, uniqueness: {scope: :is_active}, if: :is_active?
+
+  def is_active?
+    is_active == true
```

Comment:
I feel like rails gives us something like this for free, let's see if we can simplify :)

Reply:
Hmm, I'm not really sure what you're talking about. 
I tried this `validates :volunteer_id, uniqueness: {scope: :is_active}, if: { :is_active == true }`  and variations of it but it doesn't work, syntax errors. 
Am I on the right track, or are you referring to something else?

Reply:
yes that's what I'm thinking! Hmm

Reply:
Note: this was discussed and solved in slack https://rubyforgood.slack.com/archives/CVB0QJGVD/p1599271085161000